### PR TITLE
Minor fixes in RPC tutorial

### PR DESCRIPTION
**Pull request by rajarshimaitra**
Found some minor fixes in the tutorial text.

### CODE REVIEW

File: docs/_blog/Bitcoin_Core_RPC_Demo.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ edition = "2018"
 ### Setting dependencies
 Once the rust binary is compiled and running, we now need to specify the dependencies we need to work on our library.
 
-Remember that BDK provided almost everything we would need to build a wallet out of box. So we don't need any more dependencies apart from BDK. We will use another small rust crate called `dirs_next` to find our home directory and store wallet files in subfolder.
+Remember that BDK provides almost everything we would need to build a wallet out of box. So we don't need any more dependencies apart from BDK. We will use another small rust crate called [`dirs_next`](https://crates.io/crates/dirs-next) to find our home directory and store wallet files in a subfolder there.
```
Comment (notmandatory) [inline docs/_blog/Bitcoin_Core_RPC_Demo.md]:
```suggestion
Remember that BDK provides almost everything we would need to build a wallet out of the box. So we don't need any more dependencies apart from BDK. We will use another small rust crate called [`dirs_next`](https://crates.io/crates/dirs-next) to find our home directory and store wallet files in a subfolder there.
```

File: docs/_blog/Bitcoin_Core_RPC_Demo.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ With this we are now ready to add our wallet code.
...
 `"wpkh([b8b575c2/84'/1'/0'/0]tprv8icWtRzy9CWgFxpGMLSdAeE4wWyz39XGc6SwykeTo13tYm14JkVVQAf7jz8WDDarCgNJrG3aEPJEqchDWeJdiaWpS3FwbLB9SzsN57V7qxB/*)"`.
 
-This describes a SegwitV0 descriptor of a key derived at path `m/84'/1'/0'/0`. If you already have a descriptor from other sources, you can use that. Otherwise, BDK is there to hold you back. BDK can be used to generate a fresh master key with mnemonic, and then derive child keys from it given a specific path. Putting the key in a descriptor is as simple as wrapping it with a `wpkh()` string.
+This describes a SegwitV0 descriptor of a key derived at path `m/84'/1'/0'/0`. If you already have a descriptor from other sources, you can use that. Otherwise, BDK is there to hold your back. BDK can be used to generate a fresh master key with mnemonic, and then derive child keys from it given a specific path. Putting the key in a descriptor is as simple as wrapping it with a `wpkh()` string.
```
Comment (notmandatory) [inline docs/_blog/Bitcoin_Core_RPC_Demo.md]: do you mean here "BDK has your back" ? 

https://www.urbandictionary.com/define.php?term=always%20has%20your%20back

Comment (rajarshimaitra) [reply docs/_blog/Bitcoin_Core_RPC_Demo.md]: Ah right.. My bad..

File: docs/_blog/Bitcoin_Core_RPC_Demo.md (hunk 1 above)
Comment (rajarshimaitra) [reply docs/_blog/Bitcoin_Core_RPC_Demo.md]: Fixed..

**Comment by rajarshimaitra** [comment]
Thanks @notmandatory for the look. fixed them..

**Pull request merged**